### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by septa-ma** [Kubis10/CounterForMessenger#31]
Hello, I'm glad to contribute to the project.

In my opinion, it's better to make a good documentation for the project and also can use schema for getting an abstract idea of the project.

Wish you the best.

**Comment by Kubis10** [comment]
Hi! 
Thank you for contributing. What exact documentation and schema do you mean? Readme have most of it

**Comment by septa-ma** [comment]
Yes I read it, but I think maybe it can be improved by adding definition or description of functions, and can make UML diagram to have an abstract idea of the project. And of course, my suggestions are for the future.

**Comment by Kubis10** [comment]
For now, I think the project is too small, and all functions descriptions are in python file but when project will be bigger it could be helpful

**Comment by septa-ma** [comment]
Yes, I agree with you.
